shellbase-core/src/test/scala/com/sumologic/shellbase/ShellBaseTest.scala

```diff
@@ -121,18 +121,45 @@ class ShellBaseTest extends CommonWordSpec with Eventually {
       "a nested ShellCommandSet has duplicate commands" in {
```

Comment:
I am OK with the test content, but we'd rather change the test name to indicate the duplication is because of naming convention.